coprocess/bindings/python/coprocess_object_pb2.py

```diff
@@ -277,155 +277,110 @@
-      pass
-      raise NotImplementedError()
-    Dispatch.future = None
-    def DispatchEvent(self, request, timeout, metadata=None, with_call=False, protocol_options=None):
-      # missing associated documentation comment in .proto file
-      pass
-      raise NotImplementedError()
-    DispatchEvent.future = None
-
-
-  def beta_create_Dispatcher_server(servicer, pool=None, pool_size=None, default_timeout=None, maximum_timeout=None):
-    """The Beta API is deprecated for 0.15.0 and later.
-
-    It is recommended to use the GA API (classes and functions in this
-    file not marked beta) for all further purposes. This function was
-    generated only to ease transition from grpcio<0.15.0 to grpcio>=0.15.0"""
-    request_deserializers = {
-      ('coprocess.Dispatcher', 'Dispatch'): Object.FromString,
-      ('coprocess.Dispatcher', 'DispatchEvent'): Event.FromString,
-    }
-    response_serializers = {
-      ('coprocess.Dispatcher', 'Dispatch'): Object.SerializeToString,
-      ('coprocess.Dispatcher', 'DispatchEvent'): EventReply.SerializeToString,
-    }
-    method_implementations = {
-      ('coprocess.Dispatcher', 'Dispatch'): face_utilities.unary_unary_inline(servicer.Dispatch),
-      ('coprocess.Dispatcher', 'DispatchEvent'): face_utilities.unary_unary_inline(servicer.DispatchEvent),
-    }
-    server_options = beta_implementations.server_options(request_deserializers=request_deserializers, response_serializers=response_serializers, thread_pool=pool, thread_pool_size=pool_size, default_timeout=default_timeout, maximum_timeout=maximum_timeout)
-    return beta_implementations.server(method_implementations, options=server_options)
-
-
-  def beta_create_Dispatcher_stub(channel, host=None, metadata_transformer=None, pool=None, pool_size=None):
-    """The Beta API is deprecated for 0.15.0 and later.
-
-    It is recommended to use the GA API (classes and functions in this
-    file not marked beta) for all further purposes. This function was
-    generated only to ease transition from grpcio<0.15.0 to grpcio>=0.15.0"""
-    request_serializers = {
-      ('coprocess.Dispatcher', 'Dispatch'): Object.SerializeToString,
-      ('coprocess.Dispatcher', 'DispatchEvent'): Event.SerializeToString,
-    }
-    response_deserializers = {
-      ('coprocess.Dispatcher', 'Dispatch'): Object.FromString,
-      ('coprocess.Dispatcher', 'DispatchEvent'): EventReply.FromString,
-    }
-    cardinalities = {
-      'Dispatch': cardinality.Cardinality.UNARY_UNARY,
-      'DispatchEvent': cardinality.Cardinality.UNARY_UNARY,
-    }
-    stub_options = beta_implementations.stub_options(host=host, metadata_transformer=metadata_transformer, request_serializers=request_serializers, response_deserializers=response_deserializers, thread_pool=pool, thread_pool_size=pool_size)
-    return beta_implementations.dynamic_stub(channel, 'coprocess.Dispatcher', cardinalities, options=stub_options)
-except ImportError:
-  pass
+import grpc
```

Comment:
This stuff is new, did you used some experimental protobuf version to build this?

Reply:
The `Dispatcher` class looks mostly the same, the`BetaDispatcher` changed a bit, I will investigate this (shouldn't affect).

Reply:
Now it required `grpc` packages (previously it dont). 

I would say that it is better to run script with same protobuf version as we officially use now in CI.